omero_downloads.html

```diff
@@ -151,9 +161,16 @@
                                 >OMERO.importer</a> and <a
                                 href="@DOC_URL@/users/clients-overview.html#omero-editor"
                                 >OMERO.editor</a> and requires Java Version
-                                <strong>1.6</strong> or higher. OMERO.web is
-                            part of the server package, so individual users do
-                            not need to install it locally.</p>
+                                <strong>1.6</strong> or higher.</p>
+                    </li>
+                    <li>
+                        <p> Mac OS X users should use the clients matching
```

Comment:
Maybe also add a sentence here with a link to http://download.oracle.com/otn-pub/java/jdk/8u31-b13/jre-8u31-macosx-x64.dmg e.g. "MacOS X 10.7 and later should install the latest Java and use the Java7+ downloads.  MacOS X 10.6 users can only use the Java6 download".  This just makes it clearer which versions work with which MacOS X version, and how to get it without reading the docs.